pywebdriver/__init__.py

```diff
@@ -72,13 +72,16 @@
 from . import plugins  # noqa: E402
 
 # Localization
-app.config["BABEL_DEFAULT_LOCALE"] = config.get("localization", "locale")
+localization = config.get("localization", "locale")
+app.config["BABEL_DEFAULT_LOCALE"] = localization or "en_US"
```

Comment:
Hi. Thanks a lot to make this tools more international ! 

A remarks, could you imagine another default in the ``__init__.py`` file. Something like : 
- if localization defined in the config file, use it.
- else, get the localization of the current system.
- if not found, use ``en_US`` localization.

It could avoid extra useless configuration.
(Assuming that if I install pywebdriver on a german computer / raspberry-py / etc..., I expect to have the german localization, except in very specific case that could be handled in the ``config.ini`` file.)
regarding that [thread](https://stackoverflow.com/questions/3425294/how-to-detect-the-os-default-language-in-python), It seems current command could do the job.

```
>>> import locale
>>> locale.getdefaultlocale()
('fr_FR', 'UTF-8')
```

what do you think ?